app/models/user.rb

```diff
@@ -73,7 +75,11 @@ def self.new_with_session(params, session)
   def full_name
     "#{first_name} #{last_name}"
   end
-  
+
+  def can_manage?(comment)
+    comments.include?(comment)
```

Comment:
@ShevchukTania please check what SQL query it generates, if it loads all records - its a problem:
[Array of comments].include?(comment) , and if user has 10000 comments it will be slow

I expect something like code "self.tasks.where(id: task.id).exists?" for faster SQL query